app/controllers/editor.js

```diff
@@ -374,6 +383,14 @@ export default Controller.extend({
 
             return post;
         } catch (error) {
+            // trigger upgrade modal if forbidden(403) error
+            if (this._isHostLimitError(error)) {
+                this.set('post.status', prevStatus);
```

Comment:
@kevinansfield This is probably not the best way to reset post to previous state in case save fails due to host limit, as after upgrade popup the "Unsaved changes" popup is quite aggressive 😬 Do we have a standard way to rollback the post changes before attempting save like model rollback or something?

Reply:
The problem with the modal was the post being put into an error state by Ember Data due to the error occurring during a `.save()` call.

Changed the "reset" to `this.post.rollbackAttributes()`
- prevents unsaved changes modal from showing due to the post being in an error state
- all to-be-saved attributes are already in scratch values so the modal will still be shown correctly if there are real unsaved changes outside of the post status